docs/user/reference/metrics/counter.md

```diff
@@ -112,7 +123,9 @@ Increases the counter by a certain amount. If no amount is passed it defaults to
 
 ### `testGetValue`
 
-Gets the recorded value for a given counter metric.
+Gets the recorded value for a given counter metric.  
+Returns the count if data is stored.  
+Returns `null` if no data is stored.
```

Comment:
Hmmmm... what to do about languages where we don't return `null`. Like Rust where we return `None` or Firefox Desktop C++ where we return `mozilla::Nothing()` or Firefox Desktop JS where we return `undefined`...

I mean, is saying "Returns `null`" close enough for folks to get the gist? Should we pedantically list everything it could be? Will it ruin comprehension if we say "Returns `null` or some other empty/nothing value native to the specific platform"?

Hrm.

Reply:
I did briefly think about that and didn't come up with a good solution either.
I went with just `null` for brevity, but I could go with `a language-specific empty/null value`